web/thesauruses/javascript/exception_handling.json

```diff
@@ -0,0 +1,386 @@
+      "drive_disconnected_exception_type",
+      "file_no_longer_available_exception_type",
+      "file_timeout_exception"
+    ],
+    "Network Input/Output Errors/Exceptions": [
+      "base_network_exception_type",
+      "connection_refused_exception_type",
+      "connection_broken_exception_type",
+      "invalid_hostname_exception",
+      "cannot_establish_connection_exception_type",
+      "connection_timed_out_exception_type",
+      "bad_certificate_exception_type",
+      "expired_certificate_exception_type"
+    ],
+    "Operating System Errors/Exceptions": [
+      "child_process_could_not_be_created_exception_type",
+      "child_process_unexpected_ending_exception_type",
+      "process_not_found_exception_type"
+    ],
+    "Other Errors/Exception Types": [
+      "import_not_found_exception_type",
+      "keyboard_interrupt_exception_type",
+      "syntax_exception_type",
+      "uncaught_signal_exception_type"
+    ],
+    "Warnings": [
+      "base_warning_type",
+      "deprecation_warning_type"
+    ]
+  },
+  "exception_handling": {
+    "statement_to_import_exception_handling": {
+      "name": "Import statement to add exception handling",
+      "code": "class ImportError extends Error {}\nconst loadModule = async (modulePath) => {\n  try {\n    return await import(modulePath);\n  } catch (error) {\n    throw new ImportError(`Unable to import module ${modulePath}`);\n  }\n};\n"
+    },
+    "are_exception_types_functions": {
+      "name": "Are specific exception types a functions?",
+      "code": "false"
+    },
+    "are_exception_types_objects": {
+      "name": "Are specific exception types an object?",
+      "code": "true"
+    },
+    "are_exception_types_gotos": {
+      "name": "Are specific exception types a Goto statement?",
+      "code": "JavaScript has no goto statement."
+    },
+    "base_exception_type": {
+      "name": "Base exception type name",
+      "code": "Error"
+    },
+    "define_new_exception_type": {
+      "name": "Statement to define a new exception type",
+      "code": "class CustomError extends Error {}"
+    },
+    "try_or_operator": {
+      "name": "Try-Or operator",
+      "not-implemented": "true"
+    },
+    "statement_to_start_trying_code": {
+      "name": "Statement to start a block of code to try",
+      "code": "try {\n  // try statements\n}"
+    },
+    "statement_to_start_catching_exceptions": {
+      "name": "Statement to start catching exceptions",
+      "code": "} catch(error) {  \n  // catch statements\n}"
+    },
+    "statement_to_catch_generic_exception": {
+      "name": "Statement to catch a generic exception",
+      "code": "} catch(error) {  \n  // catch statements\n}"
+    },
+    "statement_to_catch_specific_exception": {
+      "name": "Statement to catch a specific exception",
+      "code": "try {\n  // try statement\n} catch (error) {\n   if ( error instanceof CustomExceptionError ) {\n    // catch statement\n  }\n}"
```

Comment:
I think the statement to catch a specific exception here might not actually exist if you have to write extra statements to determine the specific type. Since I'm trying to keep the project to be what's built into the language, this should just be "not-implemented" since it's not actually _in_ the language.

(Same for the block below)